website/pages/docs/k8s/connect/overview.mdx

```diff
@@ -412,24 +412,17 @@ If you wish to enable injection in every namespace _except_ specific namespaces,
 use `*` in the allow list to allow all namespaces and then specify the namespaces to exclude in the deny list:
 
 ```yaml
-syncCatalog:
+connectInject:
   enabled: true
   k8sAllowNamespaces: ['*']
-  k8sDenyNamespaces: ['no-sync-ns-1', 'no-sync-ns-2']
+  k8sDenyNamespaces: ['no-inject-ns-1', 'no-inject-ns-2']
 ```
 
 -> **NOTE:** The deny list takes precedence over the allow list. If a namespace
 is listed in both lists, it will **not** be synced.
 
 ~> **NOTE:** The `kube-system` and `kube-public` namespaces will never be injected.
 
-### Consul Clients Required
-
-Connect injection requires that local client agents
-are running on each Kubernetes node. These client agents must be joined to a Consul
-server cluster.
-The Consul server cluster can run either in or out of a Kubernetes cluster.
-
```

Comment:
I've removed this section because it seems to repeat what we already say in the "Installation and Configuration" section: 
> The Connect injector **also requires**
[client agents](/docs/k8s/helm#v-client) are enabled on
the node with pods that are using Connect and that
[gRPC is enabled](/docs/k8s/helm#v-client-grpc).

This page is already quite long, and I thought this was a bit redundant. Happy to add this back, if you think otherwise.